lib/alerts/components/AffectedServices.js

```diff
@@ -86,36 +111,39 @@ export default class AffectedServices extends Component {
   }
 }
 
-class ServicesHeader extends Component {
+class ServicesHeader extends Component<{entities: Array<any>}> {
```

Comment:
Should this be Array<AlertEntity> instead?  Seems like that would then negate the need for typing within each filter function.